x/examples/website-wrapper-app/.scripts/build_mobileproxy.sh

```diff
@@ -15,29 +15,24 @@
 # limitations under the License.
 
 PLATFORM="$1"
-TAG="${2:-"x/v0.0.1"}"
-OUTPUT="${3:-output}"
+OUTPUT="${2:-$(pwd)/output}"
 
 if [[ "$OUTPUT" = "/" ]] || [[ "$OUTPUT" = "*" ]]; then
   echo "Error: OUTPUT cannot be '/' or '*'. These are dangerous values."
   exit 1
 fi
 
-git clone --depth 1 --branch "$TAG" https://github.com/Jigsaw-Code/outline-sdk.git "$OUTPUT/outline-sdk"
-cd "$OUTPUT/outline-sdk/x" || exit
-go build -o "$(pwd)/out/" golang.org/x/mobile/cmd/gomobile golang.org/x/mobile/cmd/gobind
+mkdir -p "$OUTPUT/mobileproxy"
+
+go build -o "$OUTPUT/mobileproxy" golang.org/x/mobile/cmd/gomobile golang.org/x/mobile/cmd/gobind
 
 if [ "$PLATFORM" = "ios" ]; then
```

Comment:
Not needed for this PR, but you should use the `[[...]]` tests instead as per the style guide. It's safer